### PR TITLE
Deploy master

### PR DESCRIPTION
**Pull request by metallicalfa2**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

### CODE REVIEW

File: src/utils/topup.ts [hunk 1]
```diff
@@ -70,6 +70,11 @@ export const TopupProviders: { [providerName: string]: TopupProvider } = {
         label: "SOL",
         symbol: "sol",
       },
+      {
+        value: "USDC (SOL)",
```
Comment (ieow) [inline src/utils/topup.ts:74]: USC sol require token account address for moonpay topup
User may also do not have usdc token account which we might need new flow for user to create usdc token account
Suggest disable usdc topup for now

File: vue.config.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ module.exports = {
         config.plugins.push(new ProvidePlugin({process: ["process/browser"]}));
         },
     pwa: {
-        name: "Solana Wallet",
+        name: "Torus Wallet",
```
Comment (ieow) [inline vue.config.js:31]: Torus wallet for pwa?

**Review by ieow — COMMENTED** [review]
Disable usdc topup 
pwa - Torus wallet?

File: vue.config.js (hunk 2 above)
Comment (metallicalfa2) [reply vue.config.js:31]: yes

File: src/components/transfer/token-helper.ts [hunk 3]
```diff
@@ -14,7 +15,15 @@ const SOLANA_TOKEN: Partial<SolAndSplToken> = {
 // concat SOL + SPL tokens
 export const tokens = computed<Partial<SolAndSplToken>[]>(() => {
   return [
-    SOLANA_TOKEN,
+    {
+      ...SOLANA_TOKEN,
+      balance: {
+        amount: ControllerModule.solBalance.multipliedBy(LAMPORTS_PER_SOL).toString(),
```
Comment (ieow) [inline src/components/transfer/token-helper.ts:21]: Could we reduce computation by use AccountTrackerState.accounst[].lamport ?

File: src/pages/wallet/NftDetail.vue [hunk 4]
```diff
@@ -17,18 +17,17 @@ const exploreNFTS = ref<any[]>([]);
...
       const combinedCollectionObject = collections
-        .map((val) => {
-          const stats = volume.find((e) => e.collection === val.url);
+        .map((val: any) => {
+          const stats = volume.find((e: any) => e.collection === val.url);
           return { ...val, stats };
         })
-        .sort((a, b) => {
+        .sort((a: any, b: any) => {
           if (a.stats.weeklyVolume > b.stats.weeklyVolume) return -1;
```
Comment (ieow) [inline src/pages/wallet/NftDetail.vue:31]: https://github.com/torusresearch/solana-wallet/pull/189
fix sorting issue

File: src/pages/Frame.vue [hunk 5]
```diff
@@ -57,7 +58,7 @@ function startLogin() {
 }
 startLogin();
 
-const isLoggedIn = computed(() => !!ControllerModule.torus.selectedAddress);
+const isLoggedIn = computed(() => ControllerModule.torus.hasKeyPair);
```
Comment (ieow) [inline src/pages/Frame.vue:61]: fix wallet card visible during login
https://github.com/torusresearch/solana-wallet/pull/190/

**Review by metallicalfa2 — COMMENTED** [review]
lgtm

**Review by ieow — APPROVED** [review]
LGTM

**Pull request merged**